test/test_bounds.cpp

```diff
@@ -0,0 +1,134 @@
+//------------------------------------------------------------------------------
+// © 2021-2024. Triad National Security, LLC. All rights reserved.  This
+// program was produced under U.S. Government contract 89233218CNA000001
+// for Los Alamos National Laboratory (LANL), which is operated by Triad
+// National Security, LLC for the U.S.  Department of Energy/National
+// Nuclear Security Administration. All rights in the program are
+// reserved by Triad National Security, LLC, and the U.S. Department of
+// Energy/National Nuclear Security Administration. The Government is
+// granted for itself and others acting on its behalf a nonexclusive,
+// paid-up, irrevocable worldwide license in this material to reproduce,
+// prepare derivative works, distribute copies to the public, perform
+// publicly and display publicly, and to permit others to do so.
+//------------------------------------------------------------------------------
+
+#include <cstdio>
+#include <limits>
+
+#include <ports-of-call/portability.hpp>
+#include <singularity-eos/base/fast-math/logs.hpp>
+#include <singularity-eos/base/spiner_table_bounds.hpp>
+
+#ifndef CATCH_CONFIG_FAST_COMPILE
+#define CATCH_CONFIG_FAST_COMPILE
+#include <catch2/catch_test_macros.hpp>
+#endif
+
+constexpr int NGRIDS = 3;
+using Bounds = singularity::table_utils::Bounds<NGRIDS>;
```

Comment:
@Yurlungur since the `Bounds` class depends on `spiner`, can you wrap all of this in a `SINGULARITY_USE_SPINER` `#ifdef`?

I'm failing to build the tests when `SINGULARITY_USE_SPINER` is off.

I'll create an issue too.